### PR TITLE
links to similar repos in amiaopensource

### PR DESCRIPTION
**Pull request by ablwr**

### CODE REVIEW

**Comment by privatezero** [comment]
Looks good!!

**Comment by todrobbins** [comment]
🎶 What if `analog-inspection` was one of us?  🎶

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ List of open workflows and resources for A/V archiving. Please contribute!
 
 If you have never contributed to GitHub before, [here is a guide](http://ablwr.github.io/blog/2014/11/04/non-technical-persons-guide-to-becoming-an-open-source-software-contributor-via-github/)! You can also [file an issue](https://github.com/amiaopensource/open-workflows/issues/new) (which is like filling out a form) and someone else will add it.
 
+Sibling repositories: [analog-inspection](https://github.com/amiaopensource/analog-inspection) && [time-based-media-art](https://github.com/amiaopensource/time-based-media-art)
```
Comment (retokromer) [inline README.md:7]: Is there any reason for having a double `&`?

**Review by retokromer — COMMENTED** [review]
Excellent idea that we should add also to the other repositories!

File: README.md (hunk 1 above)
Comment (ablwr) [reply README.md:7]: Just programming humor-or-habit.

**Pull request merged**